src/adapters/bidderFactory.js

```diff
@@ -205,10 +206,10 @@ export function newBidder(spec) {
         switch (request.method) {
           case 'GET':
             ajax(
-              `${request.url}?${parseQueryStringParameters(request.data)}`,
+              `${request.url}?${typeof request.data === 'object' ? parseQueryStringParameters(request.data) : request.data}`,
               {
-                success: onSuccess,
-                error: onFailure
+                success: onSuccess.bind(request),
```

Comment:
`this` is basically just a sneaky way of passing the `request` parameter implicitly.

This does look like a legitimate use-case, though. To make it more explicit, could we remove the `.bind(request)` and just make the spec signature `interpretResponse(response, request)`?

If you pull `onSuccess` and `onFailure` functions inside `processRequest`, the `request` will still be in scope.